specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-03-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/notebooks/runs/{runId}": {
+      "put": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "RunNotebook_CreateRun",
+        "x-ms-examples": {
+          "Run Notebook": {
+            "$ref": "./examples/RunNotebook_Create.json"
+          }
+        },
+        "description": "Run notebook",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookRunId"
+          },
+          {
+            "name": "RunNotebookRequest",
+            "in": "body",
+            "description": "Run notebook request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RunNotebookRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted.",
+            "headers": {
+              "location": {
```

Comment:
The Azure API Guidelines state that the URL (not URI) to poll for operation status should be returned in the "Operation-Location" header.

Reply:
Please also review if you need to add `x-ms-long-running-operation` annotation.
https://azure.github.io/autorest/extensions/#x-ms-long-running-operation

Reply:
@mikekistler the API already uses location header for polling URL and I see many other existing services uses `location` as well. Is it mandatory?

Reply:
No it is not mandatory. If the convention within Synapse is to use "Location", then you should follow that convention.

Reply:
Thanks. Let's keep "location"